packages/KModal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kongponents/kmodal",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
Kongponents will auto bump/publish on merges to master so no need to manually bump.

Reply:
🤦  😢